fem/bilinearform.hpp

```diff
@@ -121,18 +125,24 @@ class BilinearForm : public Matrix
    StaticCondensation *static_cond; ///< Owned.
    Hybridization *hybridization; ///< Owned.
 
-   /** This data member allows one to specify what should be done to the
+   /** @brief This data member allows one to specify what should be done to the
        diagonal matrix entries and corresponding RHS values upon elimination of
        the constrained DoFs. */
    DiagonalPolicy diag_policy;
 
    int precompute_sparsity;
-   // Allocate appropriate SparseMatrix and assign it to mat
+
+   /// Allocate appropriate SparseMatrix and assign it to #mat
    void AllocMat();
 
+   /** @brief For partially conforming trial and/or test FE spaces, complete the
+       assembly process by performing \f$ A := P^t A P\f$ where A is the internal
+       sparse matrix and P is the conforming prolongation matrice of the
```

Comment:
I think A is here ambiguous for new users.

Reply:
```suggestion
       assembly process by performing \f$ A := P^t A P\f$ where \f$ A \f$ is the internal
       sparse matrix and \f$ P \f$ is the conforming prolongation matrix of the
```
plus some different symbol for a maybe?